.github/workflows/main.yml

```diff
@@ -59,9 +65,9 @@ jobs:
           LIB_LAPACK=$(ldconfig -p | grep liblapack.so.3 | awk '{print $4}' | head -n 1)
           echo "LD_PRELOAD=$LIB_STDCXX:$LIB_OPENBLAS:$LIB_LAPACK" >> $GITHUB_ENV
       - name: Set up MATLAB
-        uses: matlab-actions/setup-matlab@v2
+        uses: matlab-actions/setup-matlab@718d4320188c73c86eb94ce76b553cbf89778487 # v2.5.0
```

Comment:
Likewise, the need to specify particular hashes for 3rd party actions (I guess all bets are off if the core GitHub actions are compromised...), and how these get updated (can't say I'm that excited for the dependabot-driven churn, but oh well).

Reply:
Couple of thoughts:
1. We don't have to pin to hashes, we can configure zizmor to accept tags instead. But since the tj-actions thing, I'm inclined to just use hashes and let dependabot keep them updated
2. Dependabot is configured to run monthly and I'll set it to group updates into a single PR. Hopefully that will minimize the additional churn/overhead.

Reply:
While it’s producing churn, I consider it relatively minor.